share/gpodder/extensions/subscription_stats.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
+# Show publishing statistics for subscriptions.
+# Released under the same license terms as gPodder itself.
+# version 0.6 - 2023/12/07 - Nuno Dias <Nuno.Dias+gpodder@gmail.com>
+# Add Last Episode updates, sort columns and other minor changes.
+
+import datetime
+import time
+from datetime import datetime
+from time import strftime, localtime
+
+import gpodder
+from gpodder import common, download, feedcore, my, opml, player, util, youtube, config
+
+import gi  # isort:skip
+gi.require_version('Gtk', '3.0')  # isort:skip
+from gi.repository import Gtk, Pango  # isort:skip
+
+_ = gpodder.gettext
+
+__title__ = _('Subscription Statistics')
+__description__ = _('Show publishing statistics for subscriptions.')
+__only_for__ = 'gtk'
+__doc__ = 'https://gpodder.github.io/docs/extensions/subscription_stats.html'
+__authors__ = 'Brand Huntsman <http://qzx.com/mail/>'
+
+
+class gPodderExtension:
+    def __init__(self, container):
+        self.container = container
+
+    def on_ui_object_available(self, name, ui_object):
+        if name == 'gpodder-gtk':
+            self.gpodder = ui_object
+
+    def on_create_menu(self):
+        # extras menu
+        return [(_("Subscription Statistics"), self.open_dialog)]
+
+    def add_page(self, notebook, category, channels):
+        scrolled = Gtk.ScrolledWindow()
+        scrolled.set_policy(Gtk.PolicyType.NEVER, Gtk.PolicyType.AUTOMATIC)
+
+        store = Gtk.ListStore(str, str, str, int)
+        for average, name, edate, paused in channels:
+            last = strftime('%x', localtime(edate))
+            store.append([
+                ('%.1f' % round(average, 1)) if average > 0 else '?',
+                ('‖ ' if paused else '') + name, last, edate,
+            ])
+
+        tree = Gtk.TreeView(model=store)
+        scrolled.add(tree)
+
+        dayscell = Gtk.CellRendererText()
+        dayscell.set_property('xalign', 1)
+        dayscell.set_property('alignment', Pango.Alignment.RIGHT)
+        dayscolumn = Gtk.TreeViewColumn(_('Days'))
+        dayscolumn.set_sort_column_id(0)
```

Comment:
The Days column sorting does not work correctly, you need to fix it like in the `last` column by adding a column with the `average` as float to the ListStore and using it as a sort column.

Reply:
Although I can't see the problem reported I add the column and the sort is made there